studio/src/components/checks/override.tsx

```diff
@@ -222,7 +222,10 @@ export const ConfigureOverride = () => {
   );
 
   const invalidateOverrides = () => {
-    const key = createConnectQueryKey(getAllOverrides);
+    const key = createConnectQueryKey(getAllOverrides, {
+      graphName: graphContext?.graph?.name,
```

Comment:
should it default to a string? e.g., `graphContext?.graph?.name || ''`?

Reply:
It wont make a difference